src/oscar/apps/analytics/receivers.py

```diff
@@ -37,8 +39,8 @@ def _update_counter(model, field_name, filter_kwargs, increment=1):
             filter_kwargs[field_name] = increment
             model.objects.create(**filter_kwargs)
     except IntegrityError:
-        # get_or_create sometimes fails due to MySQL's weird transactions, fail
-        # silently
+        # get_or_create has a race condition (we should use upsert in supported)
+        # databases. For now just ignore these errors
```

Comment:
Ahaha. So that was the cause for that age-old comment.  

There's a misplaced `)`in the comment.